.gitlab-ci.yml

```diff
@@ -0,0 +1,133 @@
+    - TANGO_HOST=${IPADDR}:10000
+    - docker build -t cpp_tango .travis/${OS_TYPE}
+    - docker run --name cpp_tango --user root -e TANGO_HOST=${TANGO_HOST} -e BINTRAY_USER_NAME=tango-ci -e BINTRAY_API_KEY=${CI_BINTRAY_API_KEY} -e COVERALLS_REPO_TOKEN=${COVERALLS_REPO_TOKEN} --link tango_cs:tango_cs -v `pwd`:/home/tango/src -v `pwd`/idl:/home/tango/idl -v `pwd`/cppzmq:/home/tango/cppzmq -v `pwd`/coveralls-cmake:/home/tango/coveralls-cmake -v `pwd`/build-wrapper-linux-x86:/home/tango/build-wrapper-linux-x86 -dit cpp_tango
+    - .travis/install_tango_idl.sh
+    - (test ${STOCK_CPPZMQ} = "OFF" && .travis/install_cppzmq.sh) || true
+    #work around gcov ignored by sonar
+    # - sudo mkdir /home/tango && sudo mkdir /home/tango/src && sudo mount --bind `pwd` /home/tango/src
+  script:
+    - .travis/run.sh
+    - .travis/test.sh COVERALLS=OFF
+    - test ${SONAR_SCANNER} = "ON" && .travis/sonar.sh || true
+  after_script:
+    - docker stop cpp_tango
+    - docker rm cpp_tango
+    - docker stop tango_cs
+    - docker rm tango_cs
+    - docker stop mysql_db
+    - docker rm mysql_db
+    - find build -name 'core.*' -exec gzip "{}" \;
+  artifacts:
+    when: always
+    paths:
+      - build/cpp_test_suite/*/core.*
+
+debian10-release:
+  <<: *job-template
+  variables:
+    <<: *variables
+    OS_TYPE: debian10
+    CMAKE_BUILD_TYPE: Release
+
+debian10:
+  <<: *job-template
+  variables:
+    <<: *variables
+    TANGO_ENABLE_COVERAGE: "ON"
+    OS_TYPE: debian10
+  after_script:
+    - find build -name 'core.*' -exec gzip "{}" \;
+    - mkdir coverage
+    - >
+        docker exec --workdir /home/tango/src cpp_tango
+        gcovr --filter '^cppapi/' --filter '^log4tango/(?!tests/)' -j$(nproc)
+        --xml --output coverage.xml
+    - >
+        docker exec --workdir /home/tango/src cpp_tango
+        gcovr --filter '^cppapi/' --filter '^log4tango/(?!tests/)' -j$(nproc)
+        --html-details --output coverage/coverage.html
+    - >
+        docker exec --workdir /home/tango/src cpp_tango
+        gcovr --filter '^cppapi/' --filter '^log4tango/(?!tests/)' -j$(nproc)
+    - tar czf coverage.tar.gz coverage
+    - docker stop cpp_tango
+    - docker rm cpp_tango
+    - docker stop tango_cs
+    - docker rm tango_cs
+    - docker stop mysql_db
+    - docker rm mysql_db
+  artifacts:
+    when: always
+    reports:
+      cobertura: coverage.xml
+    paths:
+      - build/cpp_test_suite/*/core.*.gz
+      - coverage.xml
+      - coverage.tar.gz
+
+debian10-static:
+  <<: *job-template
+  variables:
+    <<: *variables
+    OS_TYPE: debian10
+    RUN_TESTS: "OFF"
+    BUILD_SHARED_LIBS: "OFF"
+
+debian9:
+  <<: *job-template
+  variables:
+    <<: *variables
+    OS_TYPE: debian9
+
+debian8:
+  <<: *job-template
+  variables:
+    <<: *variables
+    OS_TYPE: debian8
+    # SONAR_SCANNER: "ON"
+    # COVERALLS: "ON"
+    STOCK_CPPZMQ: "OFF"
+
+debian7:
+  <<: *job-template
+  variables:
+    <<: *variables
+    OS_TYPE: debian9
```

Comment:
Typo? debian9 -> debian7

Reply:
Fixed, thanks for noticing. Let's see if it builds: https://gitlab.com/tango-controls/test_gh_import/cppTango/-/jobs/966103473